djangoUImanager/BrainKBUIManager/settings.py

```diff
@@ -0,0 +1,169 @@
+"""
+Django settings for BrainKBUIManager project.
+
+Generated by 'django-admin startproject' using Django 4.2.11.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+import os
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-)@-kr16hschn*ezk-=_tfg(pomnf2e*qk&p#avw+^=y-t0ibsu"
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
I'd make a separate dev settings file so that this doesnt leak into other environments

Reply:
I have updated to read the key from env variable